src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -740,6 +740,12 @@ compiler.err.illegal.underscore=\
 compiler.err.illegal.dot=\
     illegal ''.''
 
+compiler.err.illegal.digit.in.binary.literal=\
+    illegal digit in an binary literal
```

Comment:
Typo: should be "a binary literal" rather than "an binary literal".

Reply:
Oops. Fixed, thanks!